lib/client.js

```diff
@@ -167,6 +169,26 @@ Client.prototype.getBroadcast = function getBroadcast(broadcastId, cb) {
   });
 };
 
+Client.prototype.listBroadcasts = function listBroadcasts(queryString, cb) {
+  var baseUrl = this.c.apiUrl + this.c.endpoints.listBroadcasts.replace(/<%apiKey%>/g, this.c.apiKey);
+  var url = queryString.length > 0 ? baseUrl + '?' + queryString.join('&') : baseUrl;
```

Comment:
It kind of bugs me that `queryString` is an array. Can we call `join()` at the end of the `listBroadcasts()` method in opentok.js?

Reply:
I was just keeping it similar to the listArchives function, but happy to do that in the `opentok.listBroadcasts` method and pass in a string instead.

Reply:
Fixed